dashboard/test/models/pd/async_workshop_handler_test.rb

```diff
@@ -0,0 +1,49 @@
+require 'test_helper'
+require 'fake_sqs/test_integration'
+
+# Launch a fake SQS service running on Localhost
+Aws.config.update(region: 'us-east-1', access_key_id: 'fake id', secret_access_key: 'fake secret')
+$fake_sqs_service = FakeSQS::TestIntegration.new(database: ':memory:',
+  sqs_endpoint: 'localhost', sqs_port: 4568)
+sleep(2) # add a sleep to fix test failures with 'RuntimeError: FakeSQS didn't start in time'
```

Comment:
Not super important, but does FakeSQS have a `wait_until_ready` or something you can use here instead?